src/nanotron/serialize/weights.py

```diff
@@ -85,6 +85,7 @@ def save_weights(model: nn.Module, parallel_context: ParallelContext, root_folde
 
             else:
                 exp_tp_pp_rank_and_size = None
+                is_expert_sharded = False
```

Comment:
why do we put is_expert_sharded = False? would be cool if we put some notes here